install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+DIR=$PWD
+
+cd $HBP/Models/CDP4_models/
+
+for p in *_poster; do
+  echo "CDP4_models/$p" >> ../_rpmbuild/models.txt
```

Comment:
slightly more elaborate would be to test if the string we want to add is not already present in the file, something like:

```bash
already_present = `grep "CDP4_models/$p" ../_rpmbuild/models.txt`
if [ -z "$already_present" ]; then;
    # we add it
fi
```